.github/workflows/ci.yaml

```
@@ -15,9 +15,22 @@ jobs:
       - uses: actions/checkout@v4
       - uses: oven-sh/setup-bun@v2
-      - name: install
-        run: bun install
-      - name: lint
-        run: bun lint:ci
-      - name: test
-        run: bun test:all
+
+      - name: Get Bun version
+        id: bun-version
+        run: echo "version=$(bun --version)" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:36: Double quote to prevent globbing and word splitting

(shellcheck)